federationapi/federationapi_test.go

```diff
@@ -273,7 +273,7 @@ func TestRoomsV3URLEscapeDoNot404(t *testing.T) {
 	cfg.Global.ServerName = gomatrixserverlib.ServerName("localhost")
 	cfg.Global.PrivateKey = privKey
 	cfg.Global.JetStream.InMemory = true
-	base := base.NewBaseDendrite(cfg, "Monolith")
+	base := base.NewBaseDendrite(cfg, "Monolith", base.DisableMetrics)
```

Comment:
This is terribly confusing because there is also `base.EnableMetrics`, however that is a struct field whereas this I guess is a static const? Can we not shadow the package name `base` with the variable `base` please!